app/pb_kits/playbook/pb_icon_body_title/_icon_body_title.jsx

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+/*eslint-disable react/no-multi-comp, flowtype/space-before-type-colon */
+
+import React from 'react'
+import Title from "../pb_title/_title.jsx";
+import Body from "../pb_body/_body.jsx";
+import Icon from "../pb_icon/_icon.jsx";
+
+type IconBodyTitleProps = {
+  body?: String,
+  className?: String,
```

Comment:
We will want to accept 1 or more classnames here. One thing that I've started to do with this is provide the type as `String | Array<String>` so that even a programmatically derived array of strings `["one", "two", "three"]` (representing class names) will work just as well as a single string "one two three".